src/main/resources/messages/monuments.properties

```diff
@@ -0,0 +1 @@
+monument.not_found=No Monument with the specified id found
```

Comment:
If we're not referencing anything specific to monuments here maybe we can change `MonumentNotFoundException` to be generic and reuse it everywhere 🤷‍♂

Reply:
Yeah, might as well. I thought maybe having the message for each record kind different but 1. What's the point and 2. Who knows how many other records we'll have anyway.